src/wp-includes/option.php

```diff
@@ -1009,7 +1022,7 @@ function add_option( $option, $value = '', $deprecated = '', $autoload = 'yes' )
 	}
 
 	if ( ! wp_installing() ) {
-		if ( 'yes' === $autoload ) {
+		if ( 'on' === $autoload || 'auto-on' === $autoload || 'yes' === $autoload ) {
```

Comment:
Just realized this is missing `auto` as well.
```suggestion
		if ( 'on' === $autoload || 'auto-on' === $autoload || 'auto' === $autoload || 'yes' === $autoload ) {
```